src/components/AttachmentPicker/index.native.js

```diff
@@ -53,6 +53,7 @@ function getImagePickerOptions(type) {
   */
 const documentPickerOptions = {
     type: [RNDocumentPicker.types.allFiles],
+    copyTo: 'cachesDirectory',
```

Comment:
Does android have a cachesDirectory? I only see it for IOS

Reply:
Documentation says:
> [iOS and Android only] copyTo:"cachesDirectory" | "documentDirectory"
https://www.npmjs.com/package/react-native-document-picker#ios-and-android-only-copytocachesdirectory--documentdirectory

I take it the option `copyTo` and the value `cachesDirectory` is available in both